core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1607,6 +1708,22 @@ class TopicChangeListener(controller: KafkaController, eventManager: ControllerE
   }
 }
 
+/**
+  * Called when broker notifies controller of disk change
+  *
+  * @param controller
```

Comment:
We should probably just remove the `@param controller` if there is no java doc for it.